src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -562,8 +565,12 @@ def augment_transactions_with_spans(
         problem_project_map = {}
         issue_occurrences = []
         occurrence_spans = set()
-        error_spans = {e["trace.span"] for e in errors if e["trace.span"]}
-        projects = {e["project.id"] for e in errors if e["trace.span"]}
+        error_spans = set()
+        projects = set()
+        for error in errors:
```

Comment:
why do we no longer have to check if the `trace.span` field exists in this loop before adding error spans / projects?

Reply:
woops good catch